src/internet_identity_interface/src/lib.rs

```diff
@@ -188,6 +188,19 @@ impl IdentityAnchorInfo {
     }
 }
 
+#[derive(Clone, Debug, CandidType, Deserialize, Eq, PartialEq)]
+pub struct Credential {
```

Comment:
I wonder whether a more specific/narrow name would be more appropriate here, as `Credential` is quite generic, and in a not-so-distant future we may add support for **verifiable credentials** (in the context of attribute sharing), so maybe we could avoid a potential future name collision.   IIUC, this is specific to a device, so maybe `DeviceCredential` would make sense?  (Note: if we change the struct name, then probably we'd also rename the related structs and functions, like `GetCredentialsResponse` or `get_credentials`). Wdyt?

Reply:
I have renamed the interface. I don't mind the longer names. @nmattia wdyt?

Reply:
Having thought about this for a bit, I think `get_webauthn_credentials` would be most accurate, if not for the fact that we also have a bit of information in there about the existence recovery phrases. But at least it would be more specific than just `device`.